proposals/community-strategy/social-media.md

```diff
@@ -0,0 +1,41 @@
+# Social media
+
+## Youtube
+
+[Youtube](https://www.youtube.com/) is an important resource for development. 
+Especially when it comes to modern web frameworks, such as React, VueJS or Angular. Some development Youtube channels 
+reach some millions of developers.
+
+### Some Youtube channels in the relevant area
+
+#### Embedded
+
+- [Andreas Spiess](https://www.youtube.com/channel/UCu7_D0o48KbfhpEohoP7YSQ)
+- [GreatScott!](https://www.youtube.com/user/greatscottlab)
+- [Make:](https://www.youtube.com/c/MAKE/featured)
+
+#### Web development
+
+- [Traversy Media](https://www.youtube.com/c/TraversyMedia/featured)
+- [freeCodeCamp.org](https://www.youtube.com/c/Freecodecamp/videos)
+- [Dev Ed](https://www.youtube.com/c/DevEd/videos)
+
+A dedicated Youtube channel with high quality videos about Web of Things would help new users to get into the topic.
+Cooperative content with big Youtube development Youtube Channel will also help to reach more users.
+
+## Stackoverflow
+
+[Stackoverflow](https://stackoverflow.com/) is the biggest question and answer site for development related questions. 
+To be present on Stackoverflow is a necessity in order to not miss questions by developer using the standard.
+
+## Discord
+
+Discord is a modern chat application. There are [several big developer chat communities](https://www.devsurvival.com/top-10-discord-servers-for-developers/).
+An own discord server is important to reach young web developer. Additionally to this cooperations with other 
+discord servers is recommended in order to reach new users and get them familiar with the standard.
+
+## Meetup
+
+[Meetups](https://www.meetup.com/) are a common way to meet other people with similar interests. 
+There are lot of different Meetups for development topics. 
+In order to reach more developer it is a good idea to be present at relevant Meetups.
```

Comment:
I went to some meetups myself, they are really nice. However, I've never organized one. I know that they can be both online/offline, so maybe it is better to start online and maybe in the future spawn some local meetups in the major cities. I know a community manager in Rome I can ping him.

Reply:
I organized some bigger meetups. But that was before Corona ^^ It is probably a bit different online. I think it is probably a good idea to just give talks at existing meetups for specific open source frameworks/libraries etc.
So the goal for these talks, hackathons etc. should be to get implemented into the library/framework.

Reply:
yeah, I participated as a speaker in a couple of Italian meetups. @egekorkan also did a great presentation at [asc 2020](https://twitter.com/egekorkan/status/1299969387437461504/photo/1). we are just starting but I think it is the right direction. 👍